src/client/common/alias_dict.cpp

```diff
@@ -175,33 +175,31 @@ void mp::AliasDict::save_dict()
         aliases_json.insert(name, definition);
     }
 
-    QTemporaryFile temp_file;
+    auto config_file_name = QString::fromStdString(aliases_file);
+
+    QDir config_path = QFileInfo(config_file_name).absoluteDir();
+    if (!config_path.exists())
+        config_path.mkpath(config_path.absolutePath());
```

Comment:
Shame `QTemporaryFile` doesn't take care of it…

Reply:
Yeah, we could wrap all this in `utils`, and do something close to `tests/temp_file.h`.

Reply:
Done, added `utils::create_temp_file_with_path()`.